File: scala-play-server/app/model/ResponseError.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ResponseError.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseError(
  httpStatus: Option[Int],
  errorCode: Option[Int],
  description: Option[String],
  documentationLink: Option[String],
  additionalInfo: Option[Map[String, List[String]]]
)

object ResponseError {
  implicit lazy val responseErrorJsonFormat: Format[ResponseError] = Json.format[ResponseError]
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseError.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open System.Collections.Generic

module ResponseError = 

  //#region ResponseError


  type ResponseError = {
    HttpStatus : int;
    ErrorCode : int;
    Description : string;
    DocumentationLink : string;
    AdditionalInfo : IDictionary<string, string[]>;
  }
  //#endregion
  